src/histograms/charts/ChartBars.ts

```diff
@@ -247,24 +247,26 @@ export class ChartBars extends AbstractChart {
           .attr('stroke-width', selectedStrokeWidth)
           .attr('height', (d) => Math.min(selectedHeadBandHeight, barsHeight - this.chartAxes.yDomain(d.value)));
       } else {
+        const maximum = max(this.histogramParams.histogramData, (d: HistogramData) => this.isValueValid(d) ? d.value : Number.MIN_VALUE);
```

Comment:
max has Number.MIN_VALUE and min has Number.MAX_VALUE, is it ok (doesn't sound logical)

Reply:
if I put `max = Number.MAX_VALUE`, the result will always be `Number.MAX_VALUE`